.gitignore

```diff
@@ -53,5 +53,5 @@ debian/files
 debian/openroad
 obj-x86_64-linux-gnu/
 
-.user-bazelrc
+user-bazelrc
```

Comment:
it is actually `user.bazelrc` according to the best practices https://bazel.build/configure/best-practices#bazelrc-file